boards/DIM/Src/App/states/App_DriveState.c

```diff
@@ -212,6 +207,38 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
         }
     }
 
+    struct ErrorList all_errors;
+    App_SharedErrorTable_GetAllErrors(error_table, &all_errors);
+
+    if (all_errors.num_errors == 0)
+    {
+        App_SevenSegDisplays_SetUnsignedBase10Value(
+            seven_seg_displays,
+            (uint32_t)App_CanRx_BMS_STATE_OF_CHARGE_GetSignal_STATE_OF_CHARGE(
+                can_rx));
+    }
+    else
+    {
+        // We want to display each error for one second, so we assume that the
+        // error set will not change much and just associate each second to
+        // a particular error. Errors that are set for less than one second may
+        // not be displayed.
```

Comment:
I think this should be `Errors that are set for less the N seconds, where N is the maximum possible number of errors, may not be displayed`?

Reply:
Ah I see what you were saying. The type of error I was thinking of that if we're displaying error A for 1 second and another error B sneaks in during that but disappears before we stop displaying error A, then error B would be not be displayed. What you're saying is also correct though.

I went with what you said since it (sort-of) encompasses the error I described.

Reply:
Did you push this change? I don't see it.

Reply:
Oops yup somehow forgot to push this small change!